### PR TITLE
make preinstalled driver path configurable

### LINKED ISSUE #1557: [AKS] make driver path configurable
## Problem: 
For existing K8s, Such as AKS env, the default driver path maybe not consist with pai driver default path. And driver install schell maybe hanging when hope to uninstall host default driver.  So it is useful to make drive path configurable for different existing K8s environmnet

## Dependency:
- Job launch script
- job-exporter
- Node manager

## Solution & Plan:
- Job launch script
code location:
    https://github.com/Microsoft/pai/blob/master/src/rest-server/src/templates/yarnContainerScript.mustache
    https://github.com/Microsoft/pai/blob/master/src/rest-server/src/models/job.js#L416

? add a configuration items at job / data var
- job-exporter
  Solution: make the driver mount path configurable
- Node manager
  Solution: make the driver mount path configurable

## Dev plan
- yanjie will create a pai/aks branch as master branch for receive all aks related pr
https://github.com/Microsoft/pai/tree/pai/aks
- some knowledge could refer yanjga/aks-test driver related change
- feature dev folk & submit pr to pai/aks branch
- If all tests passed, then cherry pick pai/aks pr to master branch.

### PR DESCRIPTION
**Pull request by xudifsd**
fix #1557

### CODE REVIEW

File: src/drivers/config/drivers.md [hunk 1]
```diff
@@ -21,18 +21,24 @@ drivers:
     set-nvidia-runtme: true
 ```
 
+Or if your cluster has already installed nvidia-driver, and do not need pai to install it
+again, then you can provide this info in your service-configuration.yaml like:
+
+```yaml
+drivers:
```
Comment (YanjieGao) [inline src/drivers/config/drivers.md]: hi @xudifsd  does this pr cherry pick to pai/aks ? i think is useful

Comment (xudifsd) [reply src/drivers/config/drivers.md]: why? you need to change the config?

Comment (YanjieGao) [reply src/drivers/config/drivers.md]: if not change could work on AKS, no need is ok

Comment (xudifsd) [reply src/drivers/config/drivers.md]: yes, the default value /usr/local/driver is where aks's driver exist, so no need to cherry-pick to aks.

Comment (YanjieGao) [reply src/drivers/config/drivers.md]: ok thanks

**Pull request merged**